numpy/add_newdocs.py

```diff
@@ -6171,11 +6171,13 @@ def luf(lamdaexpr, *args, **kwargs):
 
 add_newdoc('numpy.core.multiarray', 'dtype', ('descr',
     """
-    Array-interface compliant full description of the data-type.
+    PEP3118 interface description of the data-type.
 
     The format is that required by the 'descr' key in the
-    `__array_interface__` attribute.
+    PEP3118 `__array_interface__` attribute.
```

Comment:
`__array_interface__` is defined as part of https://docs.scipy.org/doc/numpy-1.15.1/reference/arrays.interface.html, not PEP3118.

Reply:
Hmm, OK, I may have misunderstood.